src/indicators.jl

```diff
@@ -0,0 +1,4 @@
+# Methods for porting Indicators.jl functions to TSFrame objects from TSFrames.jl package
+# See their respective documentation on Indicators.jl
+runmean(X::TSFrame, x::Symbol; args...) = TSFrame(Indicators.runmean(X[:,x]; args...))
+sma(X::TSFrame, x::Symbol; args...) = TSFrame(Indicators.sma(X[:,x]; args...))
```

Comment:
```suggestion
Indicators.runmean(X::TSFrame, x::Symbol; kwargs...) = TSFrame(Indicators.runmean(X[:,x]; kwargs...))
Indicators.sma(X::TSFrame, x::Symbol; kwargs...) = TSFrame(Indicators.sma(X[:,x]; kwargs...))
```

Probably better to be explicit when adding a method like this, things become easier.  Also, conventionally keyword arguments are splatted into `kwargs...` (not a major thing, but is easier to read).

Also, should this somehow preserve the index of `ts`?

Reply:
cool

Reply:
what did you mean by preserving the index of `ts`?